packages/console/src/components/AppContent/components/Sidebar/components/Item/index.tsx

```diff
@@ -16,12 +17,37 @@ const Item = ({ icon, titleKey, isActive = false }: Props) => {
   const { t } = useTranslation(undefined, {
     keyPrefix: 'admin_console.tabs',
   });
+  const [isContactOpen, setIsContactOpen] = useState(false);
+  const isContactUs = titleKey === 'community_support';
+
+  const handleClick: MouseEventHandler<HTMLAnchorElement> = (event) => {
+    if (!isContactUs) {
+      return;
+    }
+
+    event.preventDefault();
+    setIsContactOpen(true);
+  };
 
   return (
-    <Link to={getPath(titleKey)} className={classNames(styles.row, isActive && styles.active)}>
-      {icon && <div className={styles.icon}>{icon}</div>}
-      <div className={styles.title}>{t(titleKey)}</div>
-    </Link>
+    <>
+      <Link
+        to={getPath(titleKey)}
+        className={classNames(styles.row, isActive && styles.active)}
+        onClick={handleClick}
+      >
+        {icon && <div className={styles.icon}>{icon}</div>}
+        <div className={styles.title}>{t(titleKey)}</div>
+      </Link>
+      {isContactUs && (
```

Comment:
`Contact` component should be rendered all the time. Otherwise the component will be rendered and destroyed every time you click contact and non-contact menu item.